src/shared/sam/debugger/commands/addSamDebugConfiguration.ts

```diff
@@ -128,10 +128,15 @@ export async function addSamDebugConfiguration(
             return
         }
     } else if (type === API_TARGET_TYPE) {
-        const preloadedConfig = {
-            path: apiEvent?.Properties?.Path,
-            httpMethod: apiEvent?.Properties?.Method,
-            payload: apiEvent?.Properties?.Payload,
+        let preloadedConfig = undefined
+        if (apiEvents && apiEvents.size === 1) {
```

Comment:
why not allow  `apiEvents.size > 1` ? Or throw an error perhaps. Best to handle it somehow, rather than silently do nothing.

Reply:
This command is only adding one api event configuration (at least for now).  If it's greater than 1 somehow, then something went wrong (probably should at least log this). I thought about throwing, but I don't think we should.  With `preloadedConfig ` set to undefined, it will go to default values and still produce the config.

Reply:
> If it's greater than 1 somehow, then something went wrong (probably should at least log this). I thought about throwing, but I don't think we should.

Exactly. Something went wrong. But we should handle it somehow. Logging would be the bare minimum.